### PR TITLE
Tidy code

### PR DESCRIPTION
**Pull request by Jimmyson**
Tiding up some code. Nothing new, just a bit of streamlining.

### CODE REVIEW

**Comment by Cazzar** [comment]
Whilst some of these changes are not especially with the legacyrenamer, I can see prior code being there for readability sake, but I'll leave the decision up to other contributors

**Comment by da3dsoul** [comment]
I have only one complaint: Those parts you comment out and replace, remove them. We have version control, so reverting is easy.

**Comment by da3dsoul** [comment]
Yeah, was about to call you out on that, but was waiting to see how long it'd take.

**Comment by da3dsoul** [comment]
Damn that reviewer suggestion thing is cool. Not particularly useful since everything is going to say max and I, but it's still cool.

**Comment by Jimmyson** [comment]
> I have only one complaint: Those parts you comment out and replace, remove them. We have version control, so reverting is easy.

Just so I am clear, are you refering to "GroupStatusCollection.cs"

**Comment by da3dsoul** [comment]
There and UpdatesCollection

**Comment by Jimmyson** [comment]
I hope got that right.. 😕

File: Shoko.Server/AniDB_API/GroupStatusCollection.cs [hunk 1]
```diff
@@ -14,30 +15,14 @@ public List<Raw_AniDB_GroupStatus> Groups
...
-                int latest = 0;
-
-                foreach (Raw_AniDB_GroupStatus grp in groups)
-                {
-                    if (grp.LastEpisodeNumber > latest)
-                        latest = grp.LastEpisodeNumber;
-                }
-
-                return latest;
+                return groups.Select(grp => grp.LastEpisodeNumber).Concat(new[] {0}).Max();
```
Comment (da3dsoul) [inline Shoko.Server/AniDB_API/GroupStatusCollection.cs]: what's the point of the concat?

Comment (Jimmyson) [inline Shoko.Server/AniDB_API/GroupStatusCollection.cs]: Merge with a new empty object and the maximum result.

Comment (da3dsoul) [inline Shoko.Server/AniDB_API/GroupStatusCollection.cs]: Mhmm, I'd just use Max(a => a.Number)

Comment (da3dsoul) [inline Shoko.Server/AniDB_API/GroupStatusCollection.cs]: `groups.Max(a => a.LastEpisodeNumber)`

**Comment by da3dsoul** [comment]
Mk merging

**Comment by da3dsoul** [comment]
Btw are you on discord? Most of our dev stuff is discussed there

**Comment by Jimmyson** [comment]
I recently signed up, so yes

**Comment by da3dsoul** [comment]
Discord link is on the website

File: Shoko.Server/AniDB_API/GroupStatusCollection.cs (hunk 1 above)
Comment (hidden4003) [inline Shoko.Server/AniDB_API/GroupStatusCollection.cs]: Old version returns 1+ items, i.e. groups that don't have episodes with max (id? number?), new version always returns exactly one result. Doesn't look right to me.

Comment (da3dsoul) [inline Shoko.Server/AniDB_API/GroupStatusCollection.cs]: You're looking at an old version of the file, but the new one is syntactically equivalent.

**Comment by Jimmyson** [comment]
Do you have a channel for Server dev? I see one for WebUI and v2 dev...

**Comment by ElementalCrisis** [comment]
We have an area for only those who've contributed, I gave you access to it, you should see a channel called Dev-Talk.

**Pull request merged**